TombEngine/Game/effects/effects.cpp

```diff
@@ -1705,3 +1705,42 @@ void ProcessEffects(ItemInfo* item)
 	if (item->IsLara() && GetLaraInfo(item)->Control.WaterStatus == WaterStatus::FlyCheat)
 		item->Effect.Type = EffectType::None;
 }
+
+void TriggerAttackFlame(int x, int y, int z, Vector3 color, int size)
+{
+	auto* spark = GetFreeParticle();
```

Comment:
Adopt the use of references. Calling old functions, for now, it can be done like this: `auto& spark = *GetFreeParticle();`

Reply:
changed